server/openapi.json

```diff
@@ -0,0 +1 @@
 post by id","tags":["Posts"],"parameters":[{"in":"path","name":"postId","required":true,"description":"numeric post id","schema":{"type":"integer"}}],"responses":{"201":{"description":"Created"}}}},"/comments":{"post":{"summary":"creates new comment","description":"Takes the post data and creates a new post.","tags":["Comments"],"requestBody":{"required":true,"content":{"application/json":{"schema":{"$ref":"#/components/schemas/Comment"}}}},"responses":{"201":{"description":"Created"}}}},"/comments/{commentId}":{"delete":{"summary":"creates new comment","description":"Takes the post data and creates a new post.","tags":["Comments"],"parameters":[{"in":"path","name":"commentId","required":true,"description":"numeric comment id","schema":{"type":"integer"}}],"responses":{"201":{"description":"Created"}}}}},"components":{"schemas":{"User":{"type":"object","properties":{"email":{"type":"string","format":"email"},"email_verified":{"type":"boolean"},"username":{"type":"string"},"picture_url":{"type":"string"},"sub":{"type":"string"}},"required":["email","username","sub"],"example":{"sub":"5ebac534954b54139806c112","email":"fake@example.com","email_verified":true,"username":"fake name","picture_url":"https://asdfasdfsadfsd"}},"Space":{"type":"object","properties":{"owner":{"type":"string"},"name":{"type":"string"},"description":{"type":"string"}},"required":["owner","name","description"],"example":{"owner":"5ebac534954b54139806c112","name":"Otters are awesome","description":"space to celebrate otters"}},"Post":{"type":"object","properties":{"space_id":{"type":"integer"},"user_id":{"type":"string"},"title":{"type":"string"},"tags":{"type":"string"},"content":{"type":"string"}},"required":["space_id","user_id","title","content"],"example":{"space_id":123,"user_id":"5ebac534954b54139806c112","title":"Did you know...","content":"...that otters hold hands when they sleep?","tags":"otters, cute, fluffy"}},"Comment":{"type":"object","properties":{"user_id":{"type":"string"},"post_id":{"type":"integer"},"content":{"type":"string"}},"required":["user_id","post_id","content"],"example":{"post_id":234,"user_id":"5ebac534954b54139806c112","content":"OMG thats the cutest thing s2 🦦!!"}},"UserResponse":{"type":"object","properties":{"email":{"type":"string","x-faker":"internet.email"},"sub":{"type":"string","x-faker":"datatype.uuid"},"email_verified":{"type":"boolean"},"username":{"type":"string","x-faker":"internet.userName"},"picture_url":{"type":"string","x-faker":"image.imageUrl"}},"example":{"email":"iloveooters@gmail.com","email_verified":true,"sub":"89bd9d8d-69a6-474e-8f46-7cc8796ed151","username":"OtterLover","picture_url":"https://asdfasdfsadfsd","created_at":"2022-05-02T14:16:43.152Z"}}}},"tags":[{"name":"Users","description":"User management and retrieval"},{"name":"Spaces","description":"Spaces management and retrieval"},{"name":"Posts","description":"Posts management and retrieval"},{"name":"Comments","description":"Comments management and retrieval"}]}
```

Comment:
Ohhh !!! Now I've got it !! nice

Reply:
for mocking :) the command is `prism mock filename.json`